### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by ProgramKingAbel**
- Used a JavaScript object and array to store all of the information
- The main page is updated so that the projects section is created dynamically using the information stored in that JavaScript object on loading.
- Implemented the popup window.

### CODE REVIEW

File: pop-up.css [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+  background: #eee;
+}
+
+#modal {
+  position: fixed;
+  top: 0;
+  left: 0;
+  min-height: 100vh;
+  width: 100%;
+  background: rgba(0, 0, 0, 0.8);
```
Comment (LionRouge1) [inline pop-up.css:26]: - [ ] Please note the modal `background-color` on the Figma design, it is different from your color. Kindly check it to follow the Figma design as requested in the project requirement.

### Figma Design :point_down: 
Screenshot from 2023-04-12 15-24-10

### Your Design :point_down: 
Screenshot from 2023-04-19 13-42-25

File: pop-up.css [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+.card-info {
+  width: 100%;
+}
+
+.main-img {
+  height: auto;
+  width: 74%;
+  margin: 0 auto;
+}
+
```
Comment (LionRouge1) [inline pop-up.css:80]: - [ ] Nice work implementing the popup windows :+1: :sunglasses: However the popup images do not follow its content alignment for both the mobile and desktop version (see image). Kindly check it to follow the Figma Design as required.
Screenshot from 2023-04-19 13-35-32

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @ProgramKingAbel, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Implemented almost all the requirements:+1: 
- Followed JavaScript best practices :1st_place_medal: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: pop-up.css (hunk 1 above)
Comment (ProgramKingAbel) [reply pop-up.css]: Changed Background to match figma design

File: pop-up.css (hunk 2 above)
Comment (ProgramKingAbel) [reply pop-up.css]: Adjusted container widths to match specification

File: pop-up.css [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+  display: none;
+}
+
+.buttons {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 0.2rem 0;
+  flex-wrap: wrap;
+}
```
Comment (franclobo) [inline pop-up.css:155]: - **[OPTIONAL]** Well done implementing the required changes by the previous CR, please 🙏🏽 correct the position of the buttons to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| image | image |

**Review by franclobo — APPROVED** [review]
Hi @ProgramKingAbel, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Github flow ✔️ 
- Dynamic web page ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**